presto-docs/src/main/sphinx/security/hive.rst

```diff
@@ -0,0 +1,241 @@
+===========================
+Hive Security Configuration
+===========================
+
+The default security configuration of the :doc:`/connector/hive` uses simple
+authentication.  In this mode, the Hive connector connects to a Hadoop cluster
+that has been configured to use ``simple`` authentication. All queries are
+executed as the user who runs the Presto process, regardless of the user who
+submits the query.
+
```

Comment:
Can the first two sentences be combined with no change in meaning? 
The default security configuration of the :doc:`/connector/hive` uses simple
 +authentication to connect to a Hadoop cluster. 
(No change suggested for third sentence.)